### PR TITLE
Fix building yoga Node.js package on Windows

### LINKED ISSUE #739: Windows 10 Yarn add Issue
I tried installing yoga-layout on a windows 10 machine and I am getting this error. What can I do to fix this?

[4/4] Building fresh packages...
[-/8] ⠄ waiting...
[7/8] ⠄ yoga-layout: ERR! errno 23
[-/8] ⠄ waiting...
[-/8] ⠄ waiting...
error An unexpected error occurred: "C:\\Code\\kenleyr\\Projects\\JavaScript\\DaveyJS3\
ode_modules\\yoga-layout: Command failed.
Exit code: 23
Command: C:\\WINDOWS\\system32\\cmd.exe
Arguments: /d /s /c npm run build:node
Directory: C:\\Code\\kenleyr\\Projects\\JavaScript\\DaveyJS3\
ode_modules\\yoga-layout
Output:
> yoga-layout@1.9.3 build:node C:\\Code\\kenleyr\\Projects\\JavaScript\\DaveyJS3\
ode_modules\\yoga-layout
> npm run copy-sources && autogypi && node-gyp configure build
> yoga-layout@1.9.3 copy-sources C:\\Code\\kenleyr\\Projects\\JavaScript\\DaveyJS3\
ode_modules\\yoga-layout
> ! npm -s run is-monolithic || (rsync -r --checksum --delete ../yoga/ sources/yoga/)

'!' is not recognized as an internal or external command,
operable program or batch file.
rsync: change_dir \"/cygdrive/c/Code/kenleyr/Projects/JavaScript/DaveyJS3/node_modules/yoga-layout//../yoga\" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1178) [sender=3.1.2]
npm ERR! code ELIFECYCLE
npm ERR! errno 23
npm ERR! yoga-layout@1.9.3 copy-sources: `! npm -s run is-monolithic || (rsync -r --checksum --delete ../yoga/ sources/yoga/)`
npm ERR! Exit status 23
npm ERR!
npm ERR! Failed at the yoga-layout@1.9.3 copy-sources script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\\Users\\kenleyr\\AppData\\Roaming\
pm-cache\\_logs\\2018-03-23T15_15_07_057Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 23
npm ERR! yoga-layout@1.9.3 build:node: `npm run copy-sources && autogypi && node-gyp configure build`
npm ERR! Exit status 23

### PR DESCRIPTION
**Pull request by simast**
This PR fixes #739. It seems right now yarn/npm install always fails on windows even if you have all build prerequisites installed - due to some non cross-platform commands used in npm scripts.

### CODE REVIEW

**Comment by alecperkey** [comment]
Please consider merging

**Comment by another-guy** [comment]
@simast Thanks for creating this PR! Hopefully, it gets merged soon and resolves the build issues with the downstream dependencies such as `@react-pdf`...

**Comment by itzsaga** [comment]
👍 to getting this merged.

**Comment by kachkaev** [comment]
@davidaurelio @passy

HALP

File: javascript/binding.gyp [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-    	"includes": [
-    	    "auto.gypi",
+        "includes": [
+            "auto.gypi",
             "final-flags.gypi"
-    	],
+        ],
 
-    	"sources": [
-            "<!@(ls -1 sources/yoga/*.cpp)",
```
Comment (davidaurelio) [inline javascript/binding.gyp]: could you use `shx` here, too? The reason I am asking is that we have a growing number of lists of files that need to be kept up to date.

Comment (matthewww) [reply javascript/binding.gyp]: @simast are you still keen to get this resolved for us?

**Comment by matthewww** [comment]
@simast are you still keen to resolve this for us? 
Thanks guys for looking at this.

**Comment by kachkaev** [comment]
👋 @simast would you have a few minutes to have a look at @davidaurelio's review please? The fix is much desired by many developers on Windows laptops 🙏

**Comment by simast** [comment]
Sorry guys, I am currently on vacation, but I promise to look at the comments and will try and finish this PR once I am back in two weeks.

**Comment by andyjansson** [comment]
How's the progress on this, @simast?

**Comment by simast** [comment]
@davidaurelio I have addressed your comment and updated this PR with latest master.

**Comment by crutchcorn** [comment]
Has there been any progress on this?

**Pull request closed without merging**